### ISSUE TITLE
[Feature request] Validate version when linting

### ISSUE THREAD

**Issue by kimdv** [realm/SwiftLint#2074]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Rule Request
I have been thinking about some minimum version validating when linting, something like fastlane have in their fastfiles. When the version is less than the required version there will be an warning or error. 

The reason behind this, can be new rules in newer versions and now all developers are aware of new versions.

**Comment by marcelofabri** [comment]
You can already use `swiftlint_version` on the configuration file.

For example:

```yml
swiftlint_version: 0.30.0
```

```
$ swiftlint
Loading configuration from '.swiftlint.yml'
Currently running SwiftLint 0.25.0 but configuration specified version 0.30.0.
```

**Comment by kimdv** [comment]
Is there somehow we can get an inside Xcode error?

**Comment by marcelofabri** [comment]
Not currently. We could potentially make the script fail when that happens.

**Comment by kimdv** [comment]
Closing as PR is merged
